specs.json

```diff
@@ -1,5 +1,3 @@
 [
-  {
-    "url": "https://compat.spec.whatwg.org/"
-  }
+  "http://compat.spec.whatwg.org/"
```

Comment:
Should be `http://compat.spec.whatwg.org/`. I'll extend the lint to require this.

Reply:
should be `https://compat.spec.whatwg.org/` rather, right?

Reply:
Indeed, I missed the `s`. I guess it shows it's a good idea to add HTTPS enforcement to the linter...